cmd/kubeadm/app/phases/bootstraptoken/node/token.go

```diff
@@ -105,8 +114,12 @@ func encodeTokenSecretData(tokenID, tokenSecret string, duration time.Duration,
 		data[bootstrapapi.BootstrapTokenDescriptionKey] = []byte(description)
 	}
 	for _, usage := range usages {
-		// TODO: Validate the usage string here before
+		// Validate the usage
+		err := bootstrapapi.ValidateUsage(usage)
+		if err != nil {
+			return nil, err
```

Comment:
This will error out when the first invalid usage is checked. I'd prefer that we give the user error message including all the invalid usages the config. @luxas WDYT?